crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -474,6 +485,24 @@ impl EventTimelineItem {
             content: reply_content,
         })
     }
+
+    pub(super) fn handle(&self) -> TimelineItemHandle<'_> {
+        match &self.kind {
+            EventTimelineItemKind::Local(local) => {
+                if let Some(event_id) = local.event_id() {
+                    TimelineItemHandle::Remote(event_id)
+                } else {
+                    TimelineItemHandle::Local(
+                        local
+                            .send_handle
+                            .as_ref()
+                            .expect("EventTimelineItem::handle used in a unit test"),
```

Comment:
This message is a bit confusing to me. Does it mean that we don't expect to find a missing `send_handle` except in a unit test? Can we guarantee that? Maybe something simpler like "Unexpected missing send_handle" would work?

Reply:
> Does it mean that we don't expect to find a missing send_handle except in a unit test?
Yes

> Can we guarantee that?
We may try to sparkle generics there to mean presence or absence of the send handle, but it's overkill IMO.

> Unexpected missing send_handle
Sounds good!